app/controllers/api/v1/featured_tags_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class Api::V1::FeaturedTagsController < Api::BaseController
+  before_action -> { doorkeeper_authorize! :read, :'read:featured_tags' }, only: [:index, :show]
+  before_action -> { doorkeeper_authorize! :write, :'write:featured_tags' }, except: [:index, :show]
+
+  before_action :require_user!
+  before_action :set_featured_tags, only: :index
+  before_action :set_featured_tag, except: [:index, :create]
+
+  def index
+    render json: @featured_tags, each_serializer: REST::FeaturedTagSerializer
+  end
+
+  def show
+    render json: @featured_tag, serializer: REST::FeaturedTagSerializer
+  end
+
+  def create
+    @featured_tag = current_account.featured_tags.new(featured_tag_params)
+    @featured_tag.reset_data
+    @featured_tag.save
```

Comment:
This should use `save!` so it raises exceptions on failure.

Reply:
I see!